test/internals/reply.test.js

```diff
@@ -49,7 +49,6 @@ test('Once called, Reply should return an object with methods', t => {
   t.equal(typeof reply.status, 'function')
   t.equal(typeof reply.header, 'function')
   t.equal(typeof reply.serialize, 'function')
-  t.equal(typeof reply.getResponseTime, 'function')
```

Comment:
The `t.plan(16)` should be updated to `t.plan(15)` which I think causes the issues in the ci